src/Main.purs

```diff
@@ -6,18 +6,20 @@ import Control.Monad.Reader (ask)
 import Effect (Effect)
 import Effect.Aff (launchAff_)
 import Effect.Class (liftEffect)
-import Effect.Class.Console (log)
+import Control.Monad.Logger.Class (info)
 import Options (options)
 import Options.Applicative (execParser)
 import Printer (save) as Printer
+import Data.Map (empty)
 import Utils (catchAndKill, mkdirp, tailwindBuild)
 
 main :: Effect Unit
 main = launchAff_ $ runAppM app =<< liftEffect (execParser options)
   where
   app = do
     { config, cssInput, cssOutput, output } <- ask
+    info empty "Building Tailwind css"
```

Comment:
Simpliest kind of logging for now.

Reply:
Makes sense. Also, we'll need to pass the current log level to the js function since it void the `console.log` and `console.warn` functions for now...